deploy/charts/kubernetes-cluster-prep/README.md

```diff
@@ -0,0 +1,129 @@
+# Helm chart for Conjur Kubernetes cluster preperation
```

Comment:
Typo (should be "Preparation", and probably should use upper case for the title:
```
# Helm Chart for Conjur Kubernetes Cluster Preparation
```